api/shape/Hershey.js

```diff
@@ -1431,10 +1431,8 @@ export const toPaths = (letters) => {
     .outline();
 };
 
-export const ofSize = (size) => (text) => toPaths(text).scale(size);
+export const ofSize = (text, size) => toPaths(text).scale(size);
```

Comment:
Does this change how Hershey works? How would I use it after the change?

Reply:
Ah, it changes it to match how Text() works.

Hershey(12)('Hello') becomes Hershey('Hello', 12)